src/luaopen_milk.cpp

```diff
@@ -35,8 +35,8 @@ static int milk_quit(lua_State* L)
 	if (initialized) {
 		milk::audio_quit();
 		milk::graphics_quit();
-		milk::window_quit();
 		milk::controller_close();
```

Comment:
- Controller should be the first module closed to stick the to the reverse order of init.